third_party/txt/src/txt/paragraph.cc

```diff
@@ -871,6 +871,9 @@ Paragraph::GetMinikinFontCollectionForStyle(const TextStyle& style) {
 sk_sp<SkTypeface> Paragraph::GetDefaultSkiaTypeface(const TextStyle& style) {
   std::shared_ptr<minikin::FontCollection> collection =
       GetMinikinFontCollectionForStyle(style);
+  if (!collection) {
```

Comment:
Since you suspect that there's a better fix, maybe add a comment here that links to the bug and says that you might change to a better fix?

Reply:
On second look. I think this is fine.